src/ach/ach.js

```diff
@@ -1,9 +1,9 @@
-// 0. Get originKey
-getOriginKey().then(originKey => {
+// 0. Get clientKey
+getClientKey().then(clientKey => {
     // 1. Create an instance of AdyenCheckout
     const checkout = new AdyenCheckout({
         environment: 'test',
-        originKey: originKey // Mandatory. originKey from Customer Area
+        clientKey: clientKey // Mandatory. clientKey from Customer Area
```

Comment:
```suggestion
        clientKey // Mandatory. clientKey from Customer Area
```

Reply:
I think yes, we can leave it like that, but since this is an intro, maybe we can put it clearer as `clientKey: clientKey`.

Reply:
Hi @pabloai, I've also modified the resources to load from `checkoutshopper-test` instead of `checkoutshopper-live`!